deploy/charts/csi-driver/templates/daemonset.yaml

```diff
@@ -19,10 +19,6 @@ spec:
         - name: node-driver-registrar
           image: "{{ .Values.nodeDriverRegistrarImage.repository }}:{{ .Values.nodeDriverRegistrarImage.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          lifecycle:
```

Comment:
Just curious, why was this needed before? and why it's no longer needed..

Reply:
Please follow the issue [here](https://github.com/kubernetes-csi/node-driver-registrar/issues/21)

Reply:
thanks :man_facepalming:  :)